src/components/Loans/OpenLoans/OpenLoansControl.js

```diff
@@ -21,6 +21,8 @@ import {
 } from '../../Wrappers';
 import TableModel from './components/OpenLoansWithStaticData';
 
+/* eslint react/prop-types: "off" */
+
```

Comment:
Why are we turning off prop-types linting here? Do you have reason to believe this is a bogus warning?